Readme.md

```diff
@@ -50,26 +50,34 @@ Explorer for Endevor is included with Eclipse Che version 7.6.0 and above. Check
 
 Create a Host in Explorer for Endevor to enable it to access the repository you want to explore.
 
-### **Create a Host**
+### **Create a Profile**
 
-Create a Host in Explorer for Endevor to specify the data source you want to work with. This allows you to view and explore the selected Endevor repositories.
+Create a Profile in Explorer for Endevor to specify the data source that you want to work with. This allows you to view and explore the selected Endevor repositories.
 
+**Follow these steps:**
+
+1. Click on the + icon.
+2. Enter a name for your Profile.
+3. Enter your Endevor URL in the format `https://host:port`.
+4. (Optional) To add your mainframe credentials to your profile, enter your username and password. Adding your credentials to your profile lets you access different data sources without entering your credentials.
+5. Set profile settings to True or False to indicate that your profile should Reject or Accept connections with self-signed certificates.  
```

Comment:
Can this step be made a bit clearer? 
"Set profile settings" is vague and the functionality described in this step is specific. What actually happens? You get a dropdown prompt with true or false?
Is it "true" to reject and "false" to accept? If so make that clear because it's a bit counterintuitive.
(also, avoid "should")